### PR TITLE
add Parrot OS 4.* installation support

### PR DESCRIPTION
**Pull request by xeptore**
add support for Parrot os 4.*

### CODE REVIEW

File: deb/setup_11.x [hunk 1]
```diff
@@ -18,6 +18,7 @@ SCRSUFFIX="_11.x"
 NODENAME="Node.js 11.x"
 NODEREPO="node_11.x"
 NODEPKG="nodejs"
+DISTROID="$(lsb_release -s -i)"
```
Comment (xeptore) [inline deb/setup_11.x]: _Parrot OS_s dont have any `codename` property and return `n/a` in result of `lsb_release -s -c`, and the only remaining way to identify this OS is getting its `Distributor ID` (running `lsb_release -s -i`).

File: deb/setup_11.x [hunk 2]
```diff
@@ -195,6 +196,16 @@ check_alt() {
         echo "## You seem to be using ${1} version ${DISTRO}."
         echo "## This maps to ${3} \"${4}\"... Adjusting for you..."
         DISTRO="${4}"
+    elif [ "X${DISTROID}" == "X${1}" ]; then
```
Comment (xeptore) [inline deb/setup_11.x]: the first and the easiest way of finding _Parrot_. _(improvable)_

File: deb/setup_11.x [hunk 3]
```diff
@@ -195,6 +196,16 @@ check_alt() {
...
+        DISTROVER="$(lsb_release -r -s)"
+        if [ "X${DISTROVER:0:2}" != "X4." ]; then
```
Comment (xeptore) [inline deb/setup_11.x]: only support for version 4.* family. i couldnt find out whether other _(older)_ versions use `buster` **debian** codename or not.

**Comment by kanokwanmaneeon** [comment]
[CloudFormation.zip](https://github.com/nodesource/distributions/files/3056399/CloudFormation.zip)

**Comment by edsadr** [comment]
hey @xeptore, are you still interested in merging this? I think it will require some updates and maybe add some instructions for other people to test if it works we will be happy to merge it, closing the PR in the meantime, feel free to reopen if you want to continue

**Pull request closed without merging**